lib/app/features/chat/views/pages/new_group_modal/pages/create_group_modal.dart

```diff
@@ -166,7 +170,23 @@ class CreateGroupModal extends HookConsumerWidget {
                 ),
                 label: Text(context.i18n.group_create_create_button),
                 onPressed: () {
-                  if (formKey.currentState!.validate()) {}
+                  if (formKey.currentState!.validate()) {
+                    final newGroup = Group(
+                      id: 'new_group',
+                      link: 'https://ice.io/testers_group',
+                      name: createGroupForm.name!,
+                      type: createGroupForm.type,
+                      members: createGroupForm.members.toList(),
+                      image: ref.read(avatarProcessorNotifierProvider).mapOrNull(
+                            cropped: (file) => file.file,
+                            processed: (file) => file.file,
+                          ),
+                    );
```

Comment:
Do we know from where `Group` can come from? Maybe create provider for it?

Reply:
As for now, there is no api or any logic for this, as I know. So I made it in the same way, as new channel flow.

Reply:
Moved this stuff to the provider, which should be responsible for group creation/registration on the backend.